main.go

```diff
@@ -34,10 +34,7 @@ func twilioIncomingHandler(rw http.ResponseWriter, r *http.Request) {
 	body := strings.ToLower(r.FormValue("Body"))
 	to := r.FormValue("To")
 
-	if _, err := db.Exec("insert into message_log (from_number, to_number, body, created_at) values ($1, $2, $3, $4)", from, to, body, time.Now()); err != nil {
-		respondError(rw, err)
-		return
-	}
+	logMessage(rw, from, to, body)
```

Comment:
If the db call fails inside `logMessage` we will set the status code for the response but we don't otherwise abort the handler. Are we ok with that?